client/api/notebook.py

```diff
@@ -97,6 +101,21 @@ def save_notebook(self):
         else:
             print()
 
+def login_with_env(assignment):
```

Comment:
I think it's a good idea to move this to auth.  Keeps everything in one place, and if someone ever does use this, it'll be easy to find